doc/src/manual/multi-threading.md

```diff
@@ -18,7 +18,8 @@ The number of execution threads is controlled either by using the
 [`JULIA_NUM_THREADS`](@ref JULIA_NUM_THREADS) environment variable. When both are
 specified, then `-t`/`--threads` takes precedence.
 
-!!! compat "Julia 1.5"
+--threads {N\|auto`} Enable N threads; `auto` currently sets N to the number of local CPU threads but this might change in the future.
```

Comment:
The formatting here seems off (truncated at the beginning), and doesn't quite feel that it fits into the surrounding paragraph. Some thoughts also:

```suggestion
`--threads {N|auto}` Enable N threads; `auto` chooses a number based on the number of detected CPUs.
```